tests/Makefile

```diff
@@ -0,0 +1,22 @@
+IMAGE=odh-manifests-test
+GIT_ORG=opendatahub-io
+GIT_BRANCH=master
+SKIP_PODS_OUTPUT=
+
+all: test
+test: build run clean
+
+build:
+	podman build -t $(IMAGE) --build-arg ORG=$(GIT_ORG) --build-arg BRANCH=$(GIT_BRANCH) .
+
+run:
+	oc config view --flatten --minify > /tmp/tests-kubeconfig
+	podman run -e SKIP_PODS_OUTPUT=$(SKIP_PODS_OUTPUT) -it -v /tmp/tests-kubeconfig:/tmp/kubeconfig:z $(IMAGE)
+
+clean:
+	oc delete -n opendatahub kfdef opendatahub || true
```

Comment:
Maybe parameterize the project name, deafulting to opendatahub is probably fine though

Reply:
SInce the `opendatahub` is hardcoded elsewhere, I don't think the parameterization is necessary here, but I added the variable anyway since the value is used multiple times